discourse-mixin/config.libsonnet

```diff
@@ -0,0 +1,12 @@
+{
+  _config+:: {
+    dashboardTags: ['discourse-mixin'],
+    dashboardPeriod: 'now-1h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '1m',
+
+    // for alerts
+    alertsCritical5xxResponses: '10',  // %
+    alertsError4xxResponses: '30',  // %
```

Comment:
Both of these alerts have a "critical" severity. Can they have consistent names?